### ISSUE TITLE
Current state of Octowatch, port to Pebble SDK4

### ISSUE THREAD

**Issue by schlotzz** [jjg/octowatch#25]
The last update on Octowatch seems to be about some time ago. In the meantime, Pebble released the new SDK4. Therefore I'm building a completely new app. It's almost done. Only minor changes are being applied currently. It incorporates the following features:

- built with current SDK4
- runs on Pebble, Pebble 2, Pebble 2 SE, Pebble Time (currently working on Pebble Time 2)
- supports HTTPS (if used)
- supports HTTP Basic Auth (if used)
- uses AppGlance to display current state in menu
- shows progress bar in main view
- split C source into several files for better maintainability

I'd like to ask you for the permission to release the new app as "Octowatch2", if it's okay for you. I'm looking forward to receiving your answer.

**Comment by jjg** [comment]
Hi @schlotzz, thanks for getting in touch!  Sounds like you've done a lot of the things I've wanted to do but never got around to actually doing :)

Can you post a link to your projects Github repo so I can take a look?

**Comment by schlotzz** [comment]
Hi @jjg, thanks for your fast reply. I haven't uploaded the project on github, yet, due to some code refactoring and last polishing ;-)
In the meantime here are some screenshots from the emulator running Aplite or Basalt.

![octowatch2-screenshot1](https://cloud.githubusercontent.com/assets/4659715/20563332/6e1ff988-b188-11e6-9083-7b26daa56876.png)
![octowatch2-screenshot0](https://cloud.githubusercontent.com/assets/4659715/20563334/6e28f4b6-b188-11e6-951c-384fc67901c1.png)
![octowatch2-bw-screenshot2](https://cloud.githubusercontent.com/assets/4659715/20563333/6e26fe72-b188-11e6-9da0-6b5d3d75a765.png)

**Comment by schlotzz** [comment]
So... After putting some more work into Octowatch2 and embedding the Pebble Clay configuration framework, I did release an early alpha under https://github.com/schlotzz/octowatch2
There are still some features missing, but I'm looking forward to hearing what you think about it.

**Comment by jjg** [comment]
Hey @schlotzz, sorry for the delay.  The screenshots look good, I'll try to carve-out some time to load it up and try it out.  

I had some major changes to the UI in mind (in particular, to make it work well on Round watches) but I have no idea when I'll actually get around to implementing it, and with the Pebble/Fitbit news has kind of taken the wind out of my sails even more, so I'm comfortable with you taking Octowatch from here :)

If you feel inspired to mention me in an "easter egg", I wouldn't object ;)

**Comment by schlotzz** [comment]
Hi @jjg. Thanks for you answer and the permission to use the name Octowatch. I just released the current v1.2 of Octowatch2 in the official Pebble app store (it still works ;-) at https://apps.getpebble.com/en_US/application/58392d9900355abe6600058b
Credits have been added accordingly to the README.md at https://github.com/schlotzz/octowatch2. I'm still wondering whether to add a credits screen in the app itsself.
The acquisition of Pebble by FitBit still makes me kind of sad, but hopefully the great os and app development platform will be continued on a new level for future FitBit smartwatches... hopefully... ¯\_(ツ)_/¯
